keepalived-vip/keepalived.go

```diff
@@ -68,54 +68,73 @@ vrrp_instance vips {
   }
 }
 
-{{ range $i, $svc := .svcs }}
```

Comment:
Good idea the removal of the template of the code. Please remove the the template from the go code to avoid unnecessary duplication. The external template should check for the content to be generated (like only adding virtual_rules if there's elements in `.routes`)